java/ql/src/Security/CWE/CWE-489/DebuggableAttributeEnabled.qhelp

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>The Android manifest file defines configuration settings for Android applications.
+In this file, the <code>android:debuggable</code> attribute of the <code>application</code> element can be used to
+define whether or not the application can be debugged. When set to <code>true</code>, this attribute will allow the
+application to be debugged even when running on a device in user mode.</p>
+
+<p>When a debugger is enabled it could allow for entry points in the application or reveal sensitive information.
```

Comment:
To improve readability
```suggestion
<p>When a debugger is enabled, it could allow for entry points in the application or reveal sensitive information.
```